src/stylesheets/app.scss

```diff
@@ -0,0 +1,276 @@
+@import "base/typography";
+@import "base/vars";
+
+body{
+  margin: 0;
+  background-color: #d2e7e2; // only temporary
+}
```

Comment:
some formatting things: 

- There should be a space between selectors and `{` (e.g. `body {`)
- Put lines between selectors (e.g. between lines 7 and 8)
- It's easier to read if you break up styles into their own files and then import (e.g. `@import "components/top-fold"`)